ha/core/system_health/health_event.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
Move this file to ha/core/system_health/model/ directory. This is data model though we will not persist it.

Reply:
ok

Reply:
done